### DISCUSSION TITLE
MQTT Syntax?

### DISCUSSION THREAD

**Discussion by DakotaLarson** [uNetworking/uWebSockets.js#643] [Q&A] [answered]
Hi,
I am trying to use the documented MQTT syntax to publish updates to subscribers. Using the following code: 
```
webSocket.subscribe("test/+/d");
app.publish("test/a/d", "1");
app.publish("test/b/e", "2");
app.publish("test/c/f", "3");
app.publish("test/+/d", "4");
webSocket.subscribe("test/#");
app.publish("test/a/d", "5");
app.publish("test/b/e", "6");
app.publish("test/c/f", "7");
app.publish("test/+/d", "8");
```
I only receive "4" and "8". I would expect to see "1", "4", "5", "6", "7", "8". 

I also tried publishing these updates 1 second after the subscription call. Same results.

Is there something I am doing wrong here? Any help is appreciated.

**Comment by e3dio** [comment, marked as answer]
[v20](https://github.com/uNetworking/uWebSockets/releases/tag/v20.0.0) removed MQTT syntax, docs are now updated
